### ISSUE TITLE
Render Presets Thumbnails

### ISSUE THREAD

**Issue by OhHellNo222** [s-leger/archipack#192]
Archipack version: 1.2.83

Blender version: 2.83

Os version: win 10

Description of the issue: Hey, so I've recently found archipack and was delighted to find such an add on. I found that to get the models I needed to render the preset thumbnails so I did that the execute ran and finished no problem. now I came up against a new issue; no materials, so I installed the materials pack linked it to my blender under "Material Library" then clicked on render presets thumbnails. this time it opens the execute goes through it but it never closes remaining on my screen for hours at a time I've tried to close it in different orders, uninstalling the materials and trying again to no avail. I haven't any materials any matter what I try! 
Please Help!
Thank you. :)

**Comment by s-leger** [comment]
Hi,
The console window will not automatically close, this is a normal behavior.
Once render done - no new feedback text appear in the console - , restart blender.

**Comment by OhHellNo222** [comment]
I've tried that at least a few dozen times with many minor adjustments and my materials are still "dumb" materials. Heres my blender preference menu.
![image](https://user-images.githubusercontent.com/68819667/88489336-6421b580-cf9c-11ea-9240-7eeacd0c9ae4.png)

**Comment by s-leger** [comment]
Ensure the path for material library is absolute.
Ideally you should have a folder with archipack_matlib.blend inside, like eg "c:/addons/archipack/archipack_matlib.blend" so the path must be "c:/addons/archipack/"

**Comment by OhHellNo222** [comment]
That's exactly how I have it set up. I honestly don't know what's happening when I open the materials .blend file and put in any of the archipack models it the materials load perfectly. once I close blender the console gives me the saved to... line as I assume it should. Are there any other common mistakes that I should look into?

**Comment by s-leger** [comment]
From the picture, looks like the path is relative to current file (../../xx).
When setting the path using file chooser dialog, expand the options (small gear at right side of path chooser dialog) and ensure "relative path" is not checked.

Archipack use scene materials when found, and load from library if no material with same name is in the scene.

**Comment by OhHellNo222** [comment]
yeah, that didn't work either. Is there another method to get materials into blender? Even if it's more tedious, anything to get this to work would be much appreciated.
